### ISSUE TITLE
Updating the first editorial for the first issue.

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/journal_information#77]
@davidlmobley @dmzuckerman @dwsideriusNIST 

I will take charge of updating the editorial to use the mosts recent template version (I will work from the review template - maybe I should create an editorial template?)

What else should be changed besides the template? Things I noticed:

*  How should we give the version for editorials since our version numbering mostly talks about reviewed version and whole number versions appearing.  Is this article v2.0? Do we worry about versioning for editorials (seems maybe not, since they won't necessarily be maintained, nor are they peer reviewed?)  But on the other hand, we are making tweaks to this document.

* We should add the lessons learned category @dmzuckerman, you want to add something on this?

* Any other publishing efforts we should talk about since then?

Otherwise, I think it holds up well for taking 18 months from then to get to publication.

**Comment by dmzuckerman** [comment]
[livecoms-why-we-need-the-living-journal-of-computational-molecular-science-dmz-07jan2019.pdf](https://github.com/livecomsjournal/journal_information/files/2733829/livecoms-why-we-need-the-living-journal-of-computational-molecular-science-dmz-07jan2019.pdf)

I embedded comments and suggestions in this pdf.  Further/refined edits in PR.

One thing not in the piece now is the pedagogical emphasis and the student-reviewer requirement.  Not sure if it's strictly necessary but I wanted to mention it.

**Comment by mrshirts** [comment]
Thoughts on version numbers? Do we just call this 2.0?

**Comment by dmzuckerman** [comment]
Our editorial is unique in that it was published in 'final' form previously.  I guess the editors have to decide if it's a significant enough revision!  I'm ok with 2.0, but another choice is 1.1, which would give a hint that we didn't do anything major ... so someone who read 1.0 would not necessarily need to read 1.1.  Whatever you guys want.

**Comment by davidlmobley** [comment]
If only small changes, I'd use 1.1.

**Comment by mrshirts** [comment]
This does get away from our overall structure for paper versions that
LiveCoMS published papers are whole number version.

But then also, all research papers versions are published and have separate
DOI's.

**Comment by davidlmobley** [comment]
Yes, since this is not peer reviewed I do not feel that versions need to have whole version numbers.

**Comment by mrshirts** [comment]
How do people feel if I update to 2.0 to keep things consistent, on the publication in the first issue?

The question is whether to get a new DOI.  Might make sense.

In the future, we might not want to version editorials to avoid this issue.

**Comment by mrshirts** [comment]
I  will update on the plane tonight.

**Comment by dmzuckerman** [comment]
I don't mind what version number gets used.  The content is more important!

**Comment by davidlmobley** [comment]
I'm fine with whatever too. But just a note, @mrshirts :
> In the future, we might not want to version editorials to avoid this issue.

Not versioning things makes things MORE confusing instead of less -- then if you make updates, they are hidden. Having them versioned at least makes clear there are different versions.

**Comment by mrshirts** [comment]
Sorry, I meant that we wouldn't PRODUCE versions of editorials.  Hence not needing versioning.

**Comment by dwsideriusNIST** [comment]
Triage begins... @mrshirts @davidlmobley @dmzuckerman, you've been assigned to update the editorial.
1) Update the article template class
2) Fold revisions from @dmzuckerman into doc (see https://github.com/livecomsjournal/journal_information/issues/77#issuecomment-452043632)
2) Decide if the version number stays or not (see discussion above)

**Comment by davidlmobley** [comment]
Noted/put on tasks list.
